Makefile

```diff
@@ -37,14 +37,17 @@ golangci-lint:
 $(BUILD_DIR)/macos-amd64/$(BINARY_NAME):
 	CGO_ENABLED=0 GOARCH=amd64 GOOS=darwin go build -ldflags="$(LDFLAGS)" -o $(BUILD_DIR)/macos-amd64/$(BINARY_NAME) $(GO_BUILDFLAGS) ./cmd/admin-helper/
 
+$(BUILD_DIR)/macos-arm64/$(BINARY_NAME)-arm64:
+	CGO_ENABLED=0 GOARCH=arm64 GOOS=darwin go build -ldflags="$(LDFLAGS)" -o $(BUILD_DIR)/macos-amd64/$(BINARY_NAME)-arm64 $(GO_BUILDFLAGS) ./cmd/admin-helper/
```

Comment:
I wonder if we should suffix all binaries with `-${arch}` even the amd64 ones, or just this `arm64` binary. Worth trying this way, as hopefully this means less changes all the place.

Reply:
yeah, same thought i had, didn't want to change things in a lot of places.